mods/ctf_pvp_engine/ctf_chat/init.lua

```diff
@@ -2,9 +2,9 @@ ctf.register_on_init(function()
 	ctf.log("chat", "Initialising...")
 
 	-- Settings: Chat
-	ctf._set("chat.team_channel",          true)
-	ctf._set("chat.global_channel",        true)
-	ctf._set("chat.default",               "global")
+	ctf._set("chat.team_channel",		   true)
+	ctf._set("chat.global_channel",		   true)
+	ctf._set("chat.default",			   "global")
```

Comment:
Whitespaces are used here for aligning the tailing values, and using tabs here would mess up the alignment.